.changeset/brown-pugs-cheat.md

```diff
@@ -0,0 +1,8 @@
+---
+'rgbpp': major
```

Comment:
Why is the changeset marking `major` for the libs?

Reply:
patch?

Reply:
It's a mistake

Reply:
Updated in https://github.com/ckb-cell/rgbpp-sdk/pull/281